docs/_documentations/mdt-vsc-getting-started.md

```diff
@@ -20,7 +20,7 @@ To install Codewind for VS Code, complete the following steps:
 
 1. Install [VS Code version 1.28 or later](https://code.visualstudio.com/download).
 2. Install [Docker](https://docs.docker.com/install/) 17.06 or later. If you use Linux, you also need to install Docker Compose.
-3. Install Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind), or by searching for *Codewind* in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+3. Install Codewind by clicking [this link](vscode:extension/IBM.codewind), by searching for *Codewind* in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions), or by installing Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind).
```

Comment:
small thing but can we reorder it, i think in terms of priority it should be this link > marketplace > search in extensions view

Reply:
The link _is_ an 'install from marketplace', so maybe we can combine the two, something like

Install Codewind by clicking [this link](vscode:extension/IBM.codewind) which installs from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind), or by searching for *Codewind* in the VS Code Extensions view

Reply:
@tetchel Sure thing, have combined the two now.